charts/signoz/templates/otel-collector/schema-migrator-job.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "schemaMigrator.fullname" . }}-{{ .Release.Revision }}
+  labels:
+    jobType: schema-migrator
+spec:
+  backoffLimit: 1
```

Shouldn't this be `0` in case you want to run this job absolutely once?

```suggestion
  backoffLimit: 0
```